### PR TITLE
move & copy test helper

### PR DESCRIPTION
**Pull request by individual-it**
## Description
Test Helper to perform "MOVE" and "COPY" operations by WebDAV

## Related Issue
need to test https://github.com/owncloud/firewall/issues/411

## Motivation and Context
need to test https://github.com/owncloud/firewall/issues/411

## How Has This Been Tested?
file firewall tests

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: tests/TestHelpers/MoveCopyHelper.php [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+		$user,
+		$password,
+		$fromFileName,
+		$toFileName,
+		$headers = array(),
+		$davPathVersionToUse = 1,
+		$sourceIpAddress = null
+	) {
+		return self::copyOrMove(
+			$baseUrl, "move", $user, $password, $fromFileName, $toFileName
```
Comment (phil-davis) [inline tests/TestHelpers/MoveCopyHelper.php]: what happened to the last 3 parameters?

File: tests/TestHelpers/MoveCopyHelper.php [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+		$password,
+		$fromFileName,
+		$toFileName,
+		$headers = array(),
+		$davPathVersionToUse = 1,
+		$sourceIpAddress = null
+	) {
+		return self::copyOrMove(
+			$baseUrl, "copy", $user, $password, $fromFileName, $toFileName
+		);
```
Comment (phil-davis) [inline tests/TestHelpers/MoveCopyHelper.php]: what happened to the last 3 parameters?

**Review by phil-davis — COMMENTED** [review]
See questions.

**Comment by individual-it** [comment]
@phil-davis can you please re-review

**Review by phil-davis — APPROVED** [review]
Looking good now

**Comment by phil-davis** [comment]
Please backport to stable10, and I guess the backport PR will wait for merging until 10.0.3 is out of the way.

**Comment by individual-it** [comment]
backport to stable10 #28976

**Pull request merged**